install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -234,10 +234,10 @@ spec:
           containerPort: {{ .Values.prometheus.port }}
           hostPort: {{ .Values.prometheus.port }}
           protocol: TCP
-        {{- if .Values.proxy.prometheus.enabled }}
```

Comment:
@sayboras 

yep - indeed. refactored all occurrences of `.Values.envoy.prometheus.enabled` & `.Values.envoy.prometheus.port` to use the old "proxy value" instead of the new one.

in addition i documented the old values as deprecated.

this way we should be safe if someone is configuring and therefore passes the old values.

Reply:
@mhofstetter this change needs to be documented in the upgrade guide.